src/cli/util.js

```diff
@@ -72,23 +72,21 @@ function handleError(context, filename, error) {
   }
 
   const isParseError = Boolean(error && error.loc);
-  const isValidationError = /Validation Error/.test(error && error.message);
```

Comment:
The string "Validation Error" came from `jest-validate`, which we got rid of about 2 years ago.